components/engine_traits/src/misc.rs

```diff
@@ -54,6 +54,16 @@ pub trait StatisticsReporter<T: ?Sized> {
     fn flush(&mut self);
 }
 
+#[derive(Default)]
+pub struct RangeStats {
+    // The number of entries
+    pub num_entries: u64,
+    // The number of MVCC versions of all rows (num_entries - tombstones).
+    pub num_versions: u64,
+    // The number of rows.
+    pub num_rows: u64,
```

Comment:
What's difference between `num_entries` and `num_rows`?

Reply:
num_entries: the number of non-deleted keys of key_ts.
num_rows: the number of key without ts.